redhat-cop/infra-ansible
# WIP: add update-access for user

### What does this PR do?
This PR provides an automated way to update the root passwords on all of the inventory machines and push out the "authorized_keys" file to root user.

### How should this be tested?
 1) modify roles/update-access/test/authorized_keys 
 2) Change the inventory var 'user_name_password' 
 3) execute ansible-playbook  roles/update-access/test/test.yml -i roles/update-access/test/inventory
Verify keys and password are update on the hosts

### People to notify
cc: @oybed
